api/app/signals/apps/services/domain/filescanner.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+"""
+Check file types, both for fresh uploads and stored files.
```

Comment:
The naming of the class only implies this is a scanner used when files are uploaded, see comments on the `UploadScannerService`.